notificationbox/schema.json

```diff
@@ -25,7 +54,7 @@
           "priority": {
             "optional": true,
             "default": 1,
-            "description": "Priority ranges from 1 to 9. Default is 1.",
+            "description": "Priority ranges from PRIORITY_INFO_LOW to PRIORITY_CRITICAL_HIGH. Default is PRIORITY_INFO_LOW.",
```

Comment:
Priority ranges from PRIORITY_INFO_LOW to PRIORITY_CRITICAL_HIGH, with any combination of INFO, WARNING, CRITICAL and LOW, MEDIUM, HIGH. Default is PRIORITY_INFO_LOW.